data/catalog.yaml

```diff
@@ -535,3 +535,47 @@ works:
         expressive, and reconstructible medium. Computers will become
         substantially more powerful instruments of educational and social change
         to the extent that such an image can be realized."
+  futureofprogramming:
+    kind: post
+    authors:
+      - Stefan Lesser
+    title: "A Future Of Programming"
+    year: 2019
+    location: https://stefan-lesser.com/2019/12/24/a-future-of-programming/
+    suggestors:
+      - jryans
+    curators:
+      - 7h3kk1d
+    quotes:
+      - "Enable everyone, especially people from other domains outside
+        programming, to take advantage of computation. Empower people to build
+        their own systems and tools, giving them more of the powers that have
+        only been available to programmers through writing code, so they can
+        augment their own workflows in their own areas of expertise."
+      - "Blur the line between developers and users. Enable people to modify the
+        systems and tools they use and adapt them to their needs."
+  padplusplus:
+    kind: paper
+    authors:
+      - Benjamin B. Bederson
+      - James D. Hollan
+    title: "Pad++: a zooming graphical interface for exploring alternate interface physics"
+    year: 1994
+    location: https://dl.acm.org/doi/10.1145/192426.192435
```

Comment:
Ah yes, no particular desire to use ACM paywalled links (I think I only used it earlier in the year since it happened to be open access back then).

Perhaps let's replace this location with the direct PDF link you found, and then you could add something in the `description` field (similar to the `gtoolkit` entry) with some embedded Markdown to link to the project page, if you'd like to?

It does reveal a need for generally linking to several locations, perhaps we'll see how common that is as we add more entries over time.

Reply:
Sounds good.